### DISCUSSION TITLE
Contributors list

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#1703] [Questions & Answers]
Hi everybody, I've seen some info in the issue but I'm not sure.. so I'm asking here.

How's the list of contributors managed? Is it automatically generated or should we create a PR to update it?

Thanks!
Fab

**Comment by Flyingmana** [comment]
long story: https://github.com/OpenMage/magento-lts/issues/1126

short: currently it needs a manual PR, because we dont trust them with automated write access to the repository

an option would be to create this PRs in the future automated via github actions, but maybe not with the original nodejs project.

**Reply by sreichel** [reply]
Not "we", its just you.

**Reply by fballiano** [reply]
I'd never create a PR to add my name, that's ashaming and pretentious IMHO... that needs to be automatic.
